### ISSUE TITLE
please bugtest the hell out of the desktop mechmania site <3

### ISSUE THREAD

**Issue by bluedrops** [acm-uiuc/rp2016#153]
rolled this out yesterday after a lot of revamping

http://reflectionsprojections.com/mechmania/index.html

things i'm aware of:
- registration links lead nowhere (waiting on the registration form; my hands are tied here).
- mobile scroll handling is kinda shitty; this is because the onScroll event only fires when the page completely stops scrolling. 

bugtest the desktop site and leave notes here or create a new issue; either works

please direct all comments about the emojis to modi ;)

thanks fam <3

**Comment by sskhandek** [comment]
@bluedrops https://acmrp.typeform.com/to/PxCnjG

**Comment by bluedrops** [comment]
sujay i <3 u 
will add and PR in 10 minutes; on my way to siebel rn

**Comment by tyler-thetyrant** [comment]
![screenshot 20](https://cloud.githubusercontent.com/assets/12504517/18442085/0dedc626-78d5-11e6-9acc-47022b0ae295.png)

@bluedrops 
I think something went wrong in the last commit?
All animation shows up like that on my screen. All the text is missing as well :3

Also I feel like "Magic Sponsored by" Should be bigger.

**Comment by tyler-thetyrant** [comment]
Also,

"MechMania is a part of **Reflections | Projects.** Mechmania opens on **Friday, September 30** night after the keynote speaker at R|P. It’ll be in Siebel 2405."

Should be > Relfections|Proejctions
> Friday, September 30th, (th and the comma). *the* night after

Also I don't think we should use that many contractions in the FAQ section.
You'd -> you wouldAlso,

"MechMania is a part of **Reflections | Projects.** Mechmania opens on **Friday, September 30** night after the keynote speaker at R|P. It’ll be in Siebel 2405."

Should be > Relfections|Proejctions
> Friday, September 30th, (th and the comma). *the* night after

Also I don't think we should use that many contractions in the FAQ section.
You'd -> you would
It'll -> It will

There is two periods after yes on Free Food question. 

Missing comma after however on the last question.

3 best teams -> three best teams. 
It'll -> It will

There is two periods after yes on Free Food question. 

Missing comma after however on the last question.

3 best teams -> three best teams.

**Comment by bluedrops** [comment]
@tyler-thetyrant 
i can't seem to reproduce the animation bug you're experiencing; viewport dimensions?
